Documentation/Books/Manual/SUMMARY.md

```diff
@@ -300,6 +301,7 @@
     * [AWS](Deployment/Cloud/AWS.md)
     * [Azure](Deployment/Cloud/Azure.md)
   * [Production Checklist](Deployment/ProductionChecklist.md)
+  * [Migrating from a Single Instance to a Cluster](Deployment/MigratingSingleInstanceCluster.md)
```

Comment:
Hm, this is a pretty long text for the navigation. Maybe we should change it to _Migrating Single Instance to Cluster_?